docs/architecture.md

```
@@ -82,6 +82,31 @@ Custom adapter execution environment:
 - **Mapper Interface** (`IInfolinkMapper`)
 - **Receiver Interface** (`IInfolinkReceiver`)
 
+## Scheduling Architecture
+
+Receiving and aggregation jobs run on a persistent, clustered schedule powered by **SW-Scheduler** — a typed Quartz.NET wrapper.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)